lib/src/store.dart

```diff
@@ -85,8 +85,10 @@ class Store extends Disposable {
       throw new StateError('Store has been disposed');
     }
 
-    return _stream.listen(onData,
```

Comment:
Even if we extend Stream, we still have to implement the `listen()` method like this. The only way to get around this would be to expose the `Stream` instance directly as a property on this class.